### PR TITLE
feat: Update requirement progress format in issue table

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Update requirement progress format in issue table

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   Update requirement progress format in issue table           |
| 🇨🇳 中文    |   更新需求进度           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/dop/component-protocol/components/issue-manage/issueTable/render.go [hunk 1]
```diff
@@ -40,9 +40,16 @@ import (
 	"github.com/erda-project/erda/modules/openapi/hooks/posthandle"
 )
 
+type Multiple struct {
```
Comment (sfwn) [inline modules/dop/component-protocol/components/issue-manage/issueTable/render.go:43]: bad name.

File: modules/dop/component-protocol/components/issue-manage/issueTable/render.go [hunk 2]
```diff
@@ -40,9 +40,16 @@ import (
...
+	RenderType string        `json:"renderType,omitempty"`
+	Direction  string        `json:"direction,omitempty"`
+	Renders    []interface{} `json:"renders,omitempty"`
```
Comment (sfwn) [inline modules/dop/component-protocol/components/issue-manage/issueTable/render.go:46]: I think the actual interface type is `Progress` only.

**Review by sfwn — CHANGES_REQUESTED** [review]
Does these format config can be set in infra side?

File: modules/dop/component-protocol/components/issue-manage/issueTable/render.go (hunk 1 above)
Comment (shuofan) [reply modules/dop/component-protocol/components/issue-manage/issueTable/render.go]: done

File: modules/dop/component-protocol/components/issue-manage/issueTable/render.go (hunk 2 above)
Comment (shuofan) [reply modules/dop/component-protocol/components/issue-manage/issueTable/render.go]: done

**Comment by shuofan** [comment]
> Does these format config can be set in infra side?

yes, when we use standard table in the future

**Review by sfwn — APPROVED** [review]

**Pull request merged**